### ISSUE TITLE
MiSTer can read Wii Mote, but the cores can't

### ISSUE THREAD

**Issue by skooterblog** [MiSTer-devel/Main_MiSTer#267]
I bought a USB Wii Sensor Bar and a Wii Mote (Chinese clone). I'm aware that the motion plus version has some issues, and that the clones may have even more issues. But my problem seems odd:

- I can sync the Wii Mote just fine.
- I can map the buttons on it just fine.
- If I press F10 on the OSD, I can calibrate it just fine.
- But if I try to use it on any core, it doesn't move the crosshair at all.

I've tried different cores: NES, SMS, Genesis, SNES. I'm setting the lightgun to Joy1 as I think I'm supposed to do. But in none of them, I can move the crosshair. However, when I enter the calibration screen I realize it is working and MiSTer is reading its input just fine.

Am I doing something wrong?

This video shows my last attempts: https://youtu.be/cKqf5I_mwXs

**Comment by sorgelig** [comment]
next release will have fix for wiimote.

**Comment by Left-Empty** [comment]
I am experiencing this exact issue with all my controllers excepting keyboards. 
Controls come back after a while (30 seconds?) but sometimes won't. When the controls come back, I sometimes lose access to the OSD menu, be in trhough controllers on keyboards.
Reverting to 0703 build didn't fix it. 
I'm posting here just to get feedback in case my issue isn't tied to this one.

**Comment by felleg** [comment]
Sorry for bringing back this issue, I think the same is happening to me in a newer version!

Here is my setup:

* MiSTer OS v210711
* Wiimote connected via Bluetooth to MiSTer
* Original Wii Sensor Bar connected to Wii U for power
* (I have confirmed the Wii pointer function to work correctly on the Wii U)
* The Wii buttons are detected by MiSTer, but not the pointing function.

I have followed the instructions in this video: https://www.youtube.com/watch?v=WBoZDGn0e3U
Basically, in the NES core's input options:
- Swap joysticks: No
- Multitap: Disabled
- Serial Mode: None
- Periphery: Zapper(Joy1)
- Zapper trigger: Joystick
- Crosshairs: On

Interesting note, in the Lightgun Calibration (pressing F10 in the MiSTer menu), the pointer doesn't seem detected at all. All values are stuck at 0000.

**Comment by rautiocination** [comment]
the problem is the code just uses one wii sensor LED and there are two on the wii sensor bar.  when it loses tracking on one, it might reacquire the other LED and then all the tracking is offset by the distance between them. this is why the wiimote is designed to use at least two point minimum before tracking begins. the wiimote itself can track up to 4 for extra tracking performance, an example of such is [xwiigun](https://github.com/hifi/xwiigun) and i believe the GUN4IR also uses this as well.
